sql/item_mods.sql

```diff
@@ -39555,6 +39555,12 @@ INSERT INTO `item_mods` VALUES (19160,501,5); -- ITEM_ADDEFFECT_CHANCE: 5
 
 -- Nightfall
 INSERT INTO `item_mods` VALUES (19163,368,10); -- REGAIN: 10
+INSERT INTO `item_mods` VALUES (19163,431,2);  -- ITEM_ADDEFFECT_TYPE: DEBUFF
+INSERT INTO `item_mods` VALUES (19163,499,16); -- ITEM_SUBEFFECT: 16
+INSERT INTO `item_mods` VALUES (19163,501,5);  -- ITEM_ADDEFFECT_CHANCE: 5
+INSERT INTO `item_mods` VALUES (19163,951,28); -- ITEM_ADDEFFECT_STATUS: TERROR: 28
+INSERT INTO `item_mods` VALUES (19163,952,1);  -- ITEM_ADDEFFECT_POWER: 1
+INSERT INTO `item_mods` VALUES (19163,953,5);  -- ITEM_ADDEFFECT_DURATION: 5
```

Comment:
5 seconds is pretty long. has retails max duration before resist been measured?

Reply:
Couldn't find any info on the wikis. Used Jettatura as reference but that also has a 2 minute cooldown and a resist threshold of 0.5, so 5 seconds may be a bit long for this I agree. Can tweak it to whatever seems best.

Reply:
gonna hold for a bit and see if we can find out its peak before either asking for a change or merging as is

Reply:
![image](https://github.com/LandSandBoat/server/assets/60417494/40930e07-1b18-4547-8a28-a86151492634)
10~ seconds?
going to get some more data

Reply:
![image](https://github.com/LandSandBoat/server/assets/60417494/51720e61-61e7-464a-ac0e-43c9921a18e4)
![image](https://github.com/LandSandBoat/server/assets/60417494/b7a09ae7-6fa7-4dc3-8014-db92486c47a2)
![image](https://github.com/LandSandBoat/server/assets/60417494/a419b5d9-e974-4000-a1e5-7878562307a9)
For sure 10s base

Reply:
mind blown

got any idea how many total swings and total procs?

Reply:
Low sample size, about 168 swings, 9 procs, 5.35%ish

Reply:
even for low sample size on a single mob, bigger rate than expected.

so I guess pr need to keep its 5% proc rate and _increase_ the base duration. And someday we'll figure out why the F dark elemental seem to have crazy gobs of terror resist vs this, but nothing else tested did.

Reply:
Updated to 10 second duration.